com.unity.multiplayer.mlapi/Tests/Runtime/NetworkShowHideTests.cs

```diff
@@ -134,7 +134,8 @@ public IEnumerator NetworkShowHideTest()
             m_NetSpawnedObject2.Spawn();
             m_NetSpawnedObject3.Spawn();
 
-            yield return new WaitForSeconds(0.1f);
+            var waitUntilFrameNumber = Time.frameCount + 10;
```

Comment:
This was a temporary fix until the test is updated with these changes.
It failed on the mac build due to taking longer than 100ms.